### PR TITLE
adjust/fix default values

### LINKED ISSUE #78: JSON schema inconsistency
BOM version is optional in XML and protobuf but required in JSON.

I propose dropping the requirement from the JSON schemas as a patch release.

### PR DESCRIPTION
**Pull request by jkowalleck**
streamlined default values over multiple schema definitions types.

closes #78

### CODE REVIEW

File: schema/bom-1.5.schema.json [hunk 1]
```diff
@@ -6,8 +6,7 @@
   "$comment" : "CycloneDX JSON schema is published under the terms of the Apache License 2.0.",
   "required": [
     "bomFormat",
-    "specVersion",
-    "version"
```
Comment (jkowalleck) [inline schema/bom-1.5.schema.json:10]: `version` has a default value and is therefore optional.

File: schema/bom-1.5.xsd [hunk 2]
```diff
@@ -307,7 +307,7 @@ limitations under the License.
                     <xs:documentation>Specifies a description for the component</xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="scope" type="bom:scope" minOccurs="0" maxOccurs="1">
+            <xs:element name="scope" type="bom:scope" minOccurs="0" maxOccurs="1" default="required">
```
Comment (jkowalleck) [inline schema/bom-1.5.xsd:310]: set the same default value that was already annotated in `xs:documentation`

File: schema/bom-1.5.schema.json (hunk 1 above)
Comment (jkowalleck) [reply schema/bom-1.5.schema.json:10]: see also #78

**Comment by jkowalleck** [comment]
added label "defekt",
just like the related issue #78 was flagged as such

File: schema/bom-1.5.schema.json (hunk 1 above)
Comment (stevespringett) [reply schema/bom-1.5.schema.json:10]: specVersion should be required.

**Review by stevespringett — CHANGES_REQUESTED** [review]

File: schema/bom-1.5.schema.json (hunk 1 above)
Comment (jkowalleck) [reply schema/bom-1.5.schema.json:10]: this is true. and `specVersion` is still in the `required` set.
only `version` was removed.

File: schema/bom-1.5.schema.json [hunk 3]
```diff
@@ -6,8 +6,7 @@
...
+    "specVersion"
```
Comment (jkowalleck) [inline schema/bom-1.5.schema.json:9]: still here
re: https://github.com/CycloneDX/specification/pull/216/files/4ffa95bdae43322a9c2efac83c941443674c861f#r1241284946

**Comment by jkowalleck** [comment]
@stevespringett, could you explain what needs to be changed? 

you [mentioned](https://github.com/CycloneDX/specification/pull/216#discussion_r1241284946) in your review
> specVersion should be required.

it still is, see https://github.com/CycloneDX/specification/pull/216#discussion_r1241449544

is this enough?

**Pull request closed without merging**